.gitpod/Dockerfile

```diff
@@ -1,8 +1,8 @@
 FROM gitpod/workspace-java-17
 
 RUN bash -c ". /home/gitpod/.sdkman/bin/sdkman-init.sh && \
-    sdk install java 17.0.9-tem && \
-    sdk use java 17.0.9-tem && \
+    sdk install java 17.0.10-tem && \
+    sdk use java 17.0.10-tem && \
```

Comment:
These two commands could be simplified to `sdk env install`, allowing us to drop the versions.
This would also install mvnd, but I don't think this would hurt?

Then again I have never really used gitpod. Is the repo (`.sdkmanrc` respectively) known/visible at docker build time?
/cc @jeanphibaconnais

Reply:
Hello @famod , thanks for the suggestion. Gitpod has the same behavior that you can have on your laptop, so if `sdk env install` works, this could be apply to the gitpod file. I can check and add if